jobs/src/main/scala/dcos/metronome/jobrun/impl/JobRunServiceActor.scala

```diff
@@ -71,7 +72,8 @@ class JobRunServiceActor(
 
     val skipRun = schedule.exists(schedule => schedule.concurrencyPolicy == ConcurrencyPolicy.Forbid && runsForJob(spec.id).nonEmpty)
     if (skipRun) {
-      log.info(s"Skipping scheduled run for ${spec.id} based on concurrency policy")
+      log.warning(s"Skipping scheduled run for ${spec.id} based on concurrency policy")
+      sender() ! Failure(ConcurrentJobRunNotAllowed(spec))
```

Comment:
Funny enough, `ConcurrentJobRunNotAllowed` existed since June 2016, but was never used because we didn't reject manual triggers …

Reply:
👏